installer/dmg/scripts/postinstall

```diff
@@ -0,0 +1,166 @@
+if [[ ! -e $LOG_FILE ]]; then
+    # Create a new log file
+    touch $LOG_FILE
+else
+    # Clean up the existing file
+    > $LOG_FILE 
+fi
+
+ODBC_PATH="/Library/ODBC"
+if [[ ! -z $TARGET_VOLUME ]]; then
+    ODBC_PATH="$TARGET_VOLUME/Library/ODBC"
+fi
+INI="$ODBC_PATH/odbcinst.ini"
+info "Drivers configuration will be added to $INI"
+INSTALL_ROOT="/Library/MongoDB/AtlasSQL ODBC/"
+# Get the latest installed version
+LATEST_VERSION=$(ls "$INSTALL_ROOT/"| grep -E "[0-9]\.[0-9]\.[0-9]" | sort | cut -d '/' -f2 | tail -n 1)
+INSTALL_PATH=$INSTALL_ROOT$LATEST_VERSION
+DRIVERS_SECTION="ODBC Drivers"
+MDB="AtlasSQL ODBC $LATEST_VERSION"
+MDB_DRIVERS="$INSTALL_PATH/libatsql.dylib"
+CREATED_NEW_INI_FILE=0
+MDB_DRIVER_INSTALLED=0
+MDB_DRIVER_CONFIGURED=0
+
+# Flag the driver as installed if not already done
+flagDriversAsInstalled(){
+        if [ $MDB_DRIVER_INSTALLED == 0 ]; then
+            echo "$MDB = Installed" >> $INI
+            MDB_DRIVER_INSTALLED=1
+        fi
+}
+
+#############################################################
+### Configure the newly installed drivers in odbcinst.ini ###
+#############################################################
+# Create the ODBC directory if it doesn't exist already
+if [[ ! -e $ODBC_PATH ]]; then
+    info "Creating $ODBC_PATH directory"
+    mkdir -p $ODBC_PATH
+elif [[ ! -d $ODBC_PATH ]]; then
+     err "$ODBC_PATH already exists but is not a directory"
+     exit 1   
+fi
+
+#Update the odbcinst.ini with the latest driver path and mark it as installed
+if [[ ! -e $INI ]]; then
+    info "$INI does not exists, creating"
+    touch $INI
+    echo "[ODBC Drivers]" >> $INI
+    echo "$MDB = Installed" >> $INI
+    echo "" >> $INI
+    echo "[$MDB]" >> $INI
+    echo "Driver = $MDB_DRIVERS" >> $INI
+    echo "" >> $INI
+    $CREATED_NEW_INI_FILE=1
+fi
+
+#If there was an existing odbcinst file, update the odbcinst.ini
+# with the latest driver path
+if [[ $CREATED_NEW_INI_FILE == 0 ]]; then
+    BACK_INI="$INI.back"
+    info "Back up file $BACK_INI"
+    ODBC_DRIVERS_SECTION_EXISTS=0
+    CURRENT_SECTION="NONE"
+    PREVIOUS_SECTION="NONE"
+
+    # Make a copy of the existing ini file
+    cp $INI $BACK_INI
+    if [[ -e $BACK_INI ]]; then
+        # Clear the INI file
+        > $INI
+
+        # Read the backup odbcinst.ini.back file and recreate the new updated odbcinst.ini one
+        while IFS= read -r line
+        do
+            if [[ "$line" =~ ^\[.*\]$ ]]; then
+                PREVIOUS_SECTION=$CURRENT_SECTION
+                # Found a section
+                if grep -iqF "[ODBC Drivers]" <<< "$line"; then
+                    # Found the [ODBC Drivers] section 
+                    # where all drivers are listed
+                    ODBC_DRIVERS_SECTION_EXISTS=1
+                    CURRENT_SECTION="ODBC"
+                elif grep -iqF "[$MDB]" <<< "$line"; then
+                    # Found the [AtlasSQL Unicode ODBC {version}] section
+                    CURRENT_SECTION="UNICODE"
```

Comment:
[nit] You might want to change the name? Maybe AtlasSQLDriver? It is not incorrect though, just a little confusing I find.